src/components/SudokuGrid/Row.js

```diff
@@ -1,20 +1,14 @@
 import Cell from "./Cell";
-import { v4 as uuidv4 } from "uuid";
 
-export default function Row({
-  currentSudokuNumbers,
-  rowNumber,
-  mask,
-  onCellInRowClick,
-}) {
+export default function Row({ rowData, rowNumber, mask, onCellInRowClick }) {
```

Comment:
Cool that you are pasing now only the `rowData`. You could do the same with the mask! Just pass the `mask` for the row, not the whole mask for the whole sudoku

Reply:
Ah makes sense, I will change that:)